Project_3/style.css

```diff
@@ -0,0 +1,240 @@
+/* ==============
+OVERALL DESIGN
+================ */
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #e5eeea;
+}
+
+/* ===========
+HEADER
+============== */
+
+header {
+    background-color: #ffffff;
+    height: 100vh;
+    margin-bottom: 100px;
+    padding: 15%;
+    width: 100%;
+}
+
+h1,
+h2 {
+    font-family: 'Abril Fatface', serif;
+}
+
+h1 {
+    color: #6bcad4;
+    font-size: 100px;
+    height: 400px;
+    width: 300px;
+}
+
+h2 {
+    color: #61935f;
+    font-size: 70px;
+}
+
+
+/* ==============
+VISUAL/PSUEDO ELEMENTS 
+=================*/
+
+body::before,
+body::after {
+    background-color: #24321b;
+    content: "";
+    position: fixed;
+}
+
+/* horizontal time line */
```

Comment:
Great use of comments here -- they're especially helpful for clarifying pseudoelements 👍🏻